docs/core/tokens.md

```diff
@@ -0,0 +1,504 @@
+(both fungible and non-fungible). All tokens on Solana are effectively
+[data accounts](/docs/core/accounts#data-account) owned by the Token Program.
+
+You can find the full list of Token Program instructions
+[here](https://github.com/solana-labs/solana-program-library/blob/master/token/program/src/instruction.rs).
+
+![Token Program](/assets/docs/core/tokens/token-program.svg)
+
+A few commonly used instructions include:
+
+- [`InitializeMint`](https://github.com/solana-labs/solana-program-library/blob/master/token/program/src/processor.rs#L29):
+  Create a new mint account to represent a new type of token.
+- [`InitializeAccount`](https://github.com/solana-labs/solana-program-library/blob/master/token/program/src/processor.rs#L84):
+  Create a new token account to hold units of a specific type of token (mint).
+- [`MintTo`](https://github.com/solana-labs/solana-program-library/blob/master/token/program/src/processor.rs#L522):
+  Create new units of a specific type of token and add them to a token account.
+  This increases the supply of the token and can only be done by the mint
+  authority of the mint account.
+- [`Transfer`](https://github.com/solana-labs/solana-program-library/blob/master/token/program/src/processor.rs#L228):
+  Transfer units of a specific type of token from one token account to another.
+
+## Mint Account
+
+Tokens on Solana are uniquely identified by the address of a
+[Mint Account](https://github.com/solana-labs/solana-program-library/blob/master/token/program/src/state.rs#L18-L32)
+owned by the Token Program. This account is effectively a global counter for a
+specific token, and stores data such as:
+
+- Supply: Total supply of the token
+- Decimals: Decimal precision of the token
+- Mint authority: The account authorized to create new units of the token, thus
+  increasing the supply
+- Freeze authority: The account authorized to freeze tokens from being
+  transferred from "token accounts"
+
+![Mint Account](/assets/docs/core/tokens/mint-account.svg)
+
+The full details stored on each Mint Account include the following:
+
+```rust
+pub struct Mint {
+    /// Optional authority used to mint new tokens. The mint authority may only
+    /// be provided during mint creation. If no mint authority is present
+    /// then the mint has a fixed supply and no further tokens may be
+    /// minted.
+    pub mint_authority: COption<Pubkey>,
+    /// Total supply of tokens.
+    pub supply: u64,
+    /// Number of base 10 digits to the right of the decimal place.
+    pub decimals: u8,
+    /// Is `true` if this structure has been initialized
+    pub is_initialized: bool,
+    /// Optional authority to freeze token accounts.
+    pub freeze_authority: COption<Pubkey>,
+}
+```
+
+For reference, here is a Solana Explorer link to the USDC
+[Mint Account](https://explorer.solana.com/address/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v).
+
+### Create Mint Account
```

Comment:
Can we do this earlier as "Create a Token", explain what you did following "hey, you actually created a mint account", and do with CLI on Solana Playground/local instead?

Reply:
Goal is to show the devs what possible, then explain it. Get them their first developer success asap.

Reply:
Including both js and cli examples on the docs right now might require creating separate pages because there is not a component for creating tabs yet, and the way the page is parsed right now if you have multiple h tags with the same name it will always navigate to the first one. 

In the example below, clicking CLI or JS on Example 2, will always navigate to Example 1
```
## Example 1
### CLI
### JS

## Example 2
### CLI
### JS
```

Although for the tokens page on docs, I think it might make sense to provide context before examples since some readers might just be looking for an explanation instead of running examples.

Reply:
What I'm saying is prioritize using CLI only right now for tokens. Let's leave adding JS for when our code component supports multiple languages.

Reply:
Spent some time figuring out how to create token with the CLI. I think the JS examples would be more beneficial since they provide more context on how to interact with tokens, and act as additional references for how to build transactions/instructions. Each example already has a corresponding Solana Playground link where a reader can just press "run" to run the example. Plus, there are generally more question on Stack Exchange related to JS than how to use the CLI.

With the CLI (even on Solana Playground), it would require telling the reader to copy paste various outputs of one CLI command into the next one.

Reply:
Initially, I had the Tokens page immediately following the Transaction and Instructions page to provide more examples of building transactions and a practical example of the Solana account model.

Reply:
added cli examples